tests/scripts/test_repo_files.py

```diff
@@ -139,7 +139,7 @@ def __init__(self, *args, **kwargs):
                               capture_output=True).stdout.strip()
 
         if remote_tracking:
-            remote, remote_branch = tuple(remote_tracking.split(b'/'))
+            remote, remote_branch = tuple(remote_tracking.split(b'/', 1))
```

Comment:
A couple of small improvement suggestions here:
- I think converting to tuple is not necessary here
- the output of `run()` and `check_output` can be decoded to str right away which simplifies operating on the returned values - every further `b` may be then dropped.